### ISSUE TITLE
[Feature request] Multiple domain names (SAN) per certificate

### ISSUE THREAD

**Issue by ffittschen** [tozny/rancher-lets-encrypt#10]
Hi,
I looked through the code, but it seems like it will create a single certificate for every domain specified in `DOMAINS` in the `letsencrypt.env` file. Since lets-encrypt supports to use multiple domains within a single certificate, it would be awesome, if this could be added to `rancher-lets-encrypt`.

The use-case would be to have a single certificate for `example.com` and `www.example.com`, or `example.com` and `login.example.com`.

A possible solution would be to split at `;` per certificate and at `,` per domain:

```
DOMAINS=example.com,www.example.com;example-service.com,api.example-service.com
```

This would create two certificates:
1. `example.com` with `www.example.com` as subject alternative name
2. `example-service.com` with `api.example-service.com` as subject alternative name

When the repository supports rancher catalogs (I'll create a PR later today) The split could possibly be achieved by using a multiline question with one certificate per line and each line can list multiple certificates split by a `,`

**Edit:** Here is the PR I mentioned above: #11

**Comment by ericmann** [comment]
Looks like this was fixed with the merge of #11 (If that's not the case, please re-open with details on what's left to change).

**Comment by ffittschen** [comment]
@ericmann No, #11 only changed the repo structure to allow rancher-lets-encrypt to be used as private rancher catalog. It did not implement anything regarding multiple domain names in one certificate. This issue should be re-opened

**Comment by ericmann** [comment]
Reopened :-)
